planex/cmd/mock.py

```diff
@@ -50,6 +51,22 @@ def parse_args_or_exit(argv=None):
     return parser.parse_args(argv)
 
 
+def pty_check_call(cmd):
+    # python2.7 doesn't return the exitcode here:
+    pty.spawn(cmd)
+    # get exit status of first child
+    (pid, status) = os.waitpid(-1, 0)
+    if status != 0:
+        if os.WIFEXITED(status):
+            returncode = os.WEXITSTATUS(status)
+            print "PID %d exited with status %d" % (pid, returncode)
+        else:
+            # Killed by a signal, etc.
+            print "PID %d exited with non-zero status 0x%02x" % (pid, status)
```

Comment:
For completeness can we have

    elif os.WIFSIGNALED(status)
        signal = os.WTERMSIG(status)
        print "PID %d exited with signal %d" % (pid, signal)